cardano-cli/test/cli/genesis-initial-addr/run

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -u
+
+cwd=$(dirname "$0")
+
+# shellcheck source=/dev/null
+. "${cwd}/../core/common"
+
+# shellcheck disable=SC2154
+banner "${testname}"
+
+error=0
+
+setup_data_dir "${testname}"
+assert_file_exists "${DATA_DIR}/utxo.vkey"
+
+${CARDANO_CLI} shelley genesis key-hash \
+    --verification-key-file "${DATA_DIR}/utxo.vkey" \
+    > "${TEST_DIR}/initial-utxo-addr"
+
+assert_file_exists "${TEST_DIR}/initial-utxo-addr"
+assert_line_count 1 "${TEST_DIR}/initial-utxo-addr"
+assert_string "34257772315e97141af2d6f6ddd097d9a8923c5828434a143ab29e2d4e27f406" "${TEST_DIR}/initial-utxo-addr"
```

Comment:
We probably should add this to all tests - if  for example `--verification-key-file "${DATA_DIR}/utxo.vkey" ` will have a mistake in the filename then `"${TEST_DIR}/initial-utxo-addr"` will be created and it will pass because it will contain a one line... but it will be following message: `verification key read error: TextView IO exception at: test/cli/genesis-initial-addr/data/utxo.vkeyk Error: test/cli/genesis-initial-addr/data/utxo.vkeyk: openBinaryFile: does not exist (No such file or directory)` so I think we should also check the content and do it for other tests.

Reply:
There is something I need to fix with the way errors are reported, so that may not be necessary.

Bsically error messages should always go the `stderr` and not `stdout`.